modules/vsphere-load-balancer-multiple-subnets.adoc

```diff
@@ -0,0 +1,119 @@
+// * installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * installing_vsphere/installing-vsphere-installer-provisioned.adoc
+
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc[Installing a cluster on vSphere]
+
+[id="vsphere-load-balancer-mutliple-subnets_{context}"]
+= Configuring an external load balancer for multiple subnets
+
+You can configure an {product-title} cluster to use an external load balancer that supports multiple subnets.
+
+You must modify the default external load balancer by defining multiple network components, subnets, for vSphere. You can then distribute control plane nodes and compute nodes across multiple subnets to better manage scheduling of available pods.
+
+You do not need to specify API and Ingress static addresses for your installation program. If you choose this configuration, you must take additional actions to define network targets that accept an IP address from each referenced vSphere subnet.
+
+You can also specify the Ingress Operator to schedule pods on nodes that operate in your cluster. This configuration is useful for control plane nodes and compute nodes that you configured for running Ingress Controller pods.
+
+[IMPORTANT]
+====
+Configuring an {product-title} cluster to use an external load balancer that supports multiple subnets on vSphere is a Technology Preview feature only. To enable this feature, you must set the `featureSet:` field to  `TechPreviewNoUpgrade`. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+
+.Prerequisites
+* On your load balancer, TCP over ports 22623, 6443, 443, and 80 must be reachable by all nodes in your cluster.
+* Load balance the API port, 6443, between each of the control plane nodes.
+
+* Load balance the application ports, 443 and 80, between all the compute nodes.
+
+* On your load balancer, port 22623, which is used to serve ignition startup configurations to nodes, is not exposed outside the cluster.
+
+* Your load balancer can access the required ports on each node in your cluster. You can ensure this level of access by completing the following actions: 
+** The API load balancer can access ports 22623 and 6443 on the control plane nodes.
+** Floating IP addresses are attached to machines that use the load balancer.
```

Comment:
I'm not entirely sure what this means.  I don't think we need this.

Reply:
Removed.